### PR TITLE
Re-write of the landing page

### PR DESCRIPTION
**Pull request by mburridge**
## What?

This is a re-write of the landing page of the Block Editor Handbook.

## Why?

To make it clearer and provide some internal links to aid navigation of the handbook and highlight particularly salient areas for the reader to focus on. This also provides links to other resources, and orients the reader in the broader developer documentation space.

## How?

I typed it on a keyboard.

## Testing Instructions

n/a

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,43 +1,78 @@
...
+The editor consists of a number of key elements which are highlighted in the following graphic:
 
 ![Quick view of the block editor](https://raw.githubusercontent.com/WordPress/gutenberg/trunk/docs/assets/quick-view-of-the-block-editor.png)
 
 **Legend:**
 
-1. Block inserter
-2. Block editor content area
-3. Settings sidebar
+1. _Block inserter_
```
Comment (priethor) [inline docs/README.md]: Should it just be called `Inserter` as one can add more than blocks?

File: docs/README.md [hunk 2]
```diff
@@ -1,43 +1,78 @@
...
+2. _Block editor content area_
+3. _Settings sidebar_
```
Comment (priethor) [inline docs/README.md]: This should probably be called `Sidebar Inspector` or `Settings Sidebar`. It also contains page settings and global styles, not just block settings.

File: docs/README.md [hunk 3]
```diff
@@ -1,43 +1,78 @@
...
+
+The block editor is designed with progressive enhancement in mind, meaning that it is backward compatible with all legacy content. Blocks also offer enhanced editing and format controls.
+
+Not only does the block editor enhance the editing experience by offering visual drag-and-drop content creation tools, it is also a powerful developer platform with a rich feature-set of APIs that allow it to be manipulated and extended in a multitude of different ways.
+
+### The Gutenberg project
 
-Using a system of Blocks to compose and format content, the new block-based editor is designed to create rich, flexible layouts for websites and digital products. Content is created using blocks instead of freeform text with inserted media, embeds and Shortcodes (there's a Shortcode block, though).
+“[Gutenberg](https://github.com/WordPress/gutenberg/)” is the codename for the project to develop the block editor. It aims to revolutionize the entire publishing experience, just as Johannes Gutenberg revolutionized the printed word back in the 15th century.
```
Comment (priethor) [inline docs/README.md]: `Publishing experience` seems out of place in the context of the editor. It is the only reference to publishing in this document and doesn't fit. Maybe "the WordPress editing experience"?

File: docs/README.md [hunk 4]
```diff
@@ -1,43 +1,78 @@
...
+- **[Explanations](https://developer.wordpress.org/block-editor/explanations/)** - this section enables you to go deeper and reinforce your practical knowledge with a theoretical understanding of the [Architecture](https://developer.wordpress.org/block-editor/explanations/architecture/) of the block editor. There’s also a useful [Glossary of terms](https://developer.wordpress.org/block-editor/explanations/glossary/), and the [FAQs](https://developer.wordpress.org/block-editor/explanations/faq/) should answer any outstanding questions you may have.
```
Comment (priethor) [inline docs/README.md]: At first glance, it is hard to understand what Explanations are, considering there are already fundamentals linked in the first section.

As far as I understand, this document follows the [Divio Documentation System](https://documentation.divio.com/). Therefore, wouldn't the content of getting started belong to `How-tos` (how to set a dev environment) and `Explanations` (fundamentals of blocks)?

File: docs/README.md [hunk 5]
```diff
@@ -1,43 +1,78 @@
...
+- **[Contributor Guide](https://developer.wordpress.org/block-editor/contributors/)e** - Gutenberg is open source software and anyone is welcome to contribute to the project. This section details how to contribute and can help you choose in which way you want to contribute, whether that be with [code](https://developer.wordpress.org/block-editor/contributors/code/), with [design](https://developer.wordpress.org/block-editor/contributors/design/), with [documentation](https://developer.wordpress.org/block-editor/contributors/documentation/), or in some other way.
 
-### Create a Block Tutorial
+### Further resources
+This handbook should be considered the canonical resource for all things related to block development. However there are other resources that can help you.
+#### Blog
+The [WordPress Developer Blog](https://developer.wordpress.org/news/) is an ever-growing resource of technical articles covering specific topics related to block development and covering a wide variety of use cases. The blog is also an excellent way to keep up with the latest developments in WordPress.
+#### Courses
```
Comment (priethor) [inline docs/README.md]: I would try to reduce the `Further Resources` section, which is nearly as long as the main one. For example, the `Courses` subsection is too detailed, considering we are on the Handbook landing page.

**Review by priethor — APPROVED** [review]
Thanks for addressing the feedback! I'd rather merge these changes now and iterate as soon as the [discussion about the Handbook structure](https://github.com/WordPress/gutenberg/discussions/48998) throws some clarity.

**Review by juanmaguitar — APPROVED** [review]

**Comment by ryanwelcher** [comment]
Closes: https://github.com/WordPress/gutenberg/issues/35905

**Pull request merged**